src/components/@shared/VerifiedBadge/index.tsx

```diff
@@ -34,32 +53,26 @@ export default function VerifiedBadge({
 
   return (
     <div className={styles.container}>
-      <div className={styleClasses}>
-        <div className={styles.mainLabel}>
-          {isLoading ? (
-            <div className={styles.loader}>
-              <Loader />
-            </div>
-          ) : isInvalid ? (
-            <Cross />
-          ) : (
-            <VerifiedPatch />
-          )}
-          <span>{text}</span>
-        </div>
-        <div className={styles.details}>
-          {apiVersion && (
-            <span className={styles.apiVersion}>
-              version: {formattedApiVersion}
-            </span>
-          )}
-          {timestamp && (
-            <span className={styles.lastVerified}>
-              last check: <Time date={new Date().toString()} relative />
-            </span>
-          )}
+      {isLoading ? (
+        <Loader message="Verifying Service Credential" />
+      ) : (
+        <div className={styleClasses}>
+          <Badge isValid={isValid} label="Service Credential" />
+          <Badge isValid={idMatch} label="Credential ID match" />
```

Comment:
suggestion: change label to something like "Verified Service", as this will most likely be a more elaborate check in the future (other than just ID=DID)

Reply:
Also happy with that, but since only the credential is verified and not the service (today) I would not do it now.